src/System.Collections.Concurrent/tests/ConcurrentQueueTests.cs

```diff
@@ -17,120 +17,7 @@ public class ConcurrentQueueTests : ProducerConsumerCollectionTests
         protected override bool IsEmpty(IProducerConsumerCollection<int> pcc) => ((ConcurrentQueue<int>)pcc).IsEmpty;
         protected override bool TryPeek(IProducerConsumerCollection<int> pcc, out int result) => ((ConcurrentQueue<int>)pcc).TryPeek(out result);
         protected override bool ResetImplemented => false;
-
-        [Fact]
-        public void Ctor_NoArg_ItemsAndCountMatch()
```

Comment:
Oh I see, these are the changes from you rebasing your branch onto the latest master. Weird.

Thanks for the contribution, I merged this PR.